streamparse/cmdln.py

```diff
@@ -79,8 +83,11 @@ def main():
     if args["run"]:
         time = int(args["--time"])
         par = int(args["--par"])
+        ackers = int(args["--ackers"]) if args.get("--ackers") else par
```

Comment:
If you're using `args.get` here anyway, you might as well just use par as a default value:

```python
ackers = int(args.get("--ackers", par))
workers = int(args.get("--workers", par))
```